etc/scripts/openhub_scraper.py

```diff
@@ -0,0 +1,36 @@
+# A script to extract openhub license names, their urls, and save it in a
+# json file
+import json
+import urllib2
+from bs4 import BeautifulSoup
```

Comment:
Please sort your imports in two blocks and add one blank line between stdlib and non-stdlib

Reply:
Addressed !